modules/dastool/dastool/meta.yml

```diff
@@ -0,0 +1,82 @@
+name: dastool_dastool
+description: DAS Tool binning step.
+keywords:
+  - binning
+  - das tool
+  - table
+  - de novo
+  - bins
+  - contigs
+  - assembly
+tools:
+  - dastool:
+      description: |
+        DAS Tool is an automated method that integrates the results
+        of a flexible number of binning algorithms to calculate an optimized, non-redundant
+        set of bins from a single assembly.
+
+      homepage: https://github.com/cmks/DAS_Tool
+      documentation: https://github.com/cmks/DAS_Tool
+      tool_dev_url: https://github.com/cmks/DAS_Tool
+      doi: "10.1038/s41564-018-0171-1"
+      licence: ['BSD']
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - contigs:
+      type: file
+      description: fasta file
+      pattern: "*.{fa,fas,fasta}"
+  - bins:
+      type: file
+      description: "Scaffolds2bin tabular file generated with dastool/scaffolds2bin"
+      pattern: "*.scaffolds2bin.tsv"
+  - proteins:
+      type: file
+      description: Predicted proteins in prodigal fasta format (>scaffoldID_geneNo)
+      pattern: "*.{fa,fas,fasta}"
```

Comment:
Should these gzipped as well? Or rather account for gzipped/ungzipped possible input?